### ISSUE TITLE
Getting and setting `specFilter` directly from `Env` is deprecated in Jasmine 3.3.0

### ISSUE THREAD

**Issue by greg-fix** [mnasyrov/karma-jasmine-spec-tags#1]
First of all, thank you for creating this helpful utility!

**Issue:**

With Jasmine 3.3.0 getting and setting `specFilter` directly from `Env` is deprecated. This method was deprecated in the following commit: https://github.com/jasmine/jasmine/commit/92d0882a3202e51a866c7b0dfed5393a0e9fe338. When using this utitlity and version 3.3.0 of Jasmine or higher we get the following deprecation warnings:

```
'DEPRECATION:', 'Getting specFilter directly from Env is deprecated, please check the specFilter option from `configuration`'
'DEPRECATION:', 'Setting specFilter directly on Env is deprecated, please use the specFilter option in `configure`'
```

**Proposed Fix:**

Update lines 20 - 21 of `client.js` from:
```
    var specFilterFnDelegate = jasmineEnv.specFilter;
    jasmineEnv.specFilter = customSpecFilter;
```
to:
```
    var specFilterFnDelegate = jasmineEnv.configuration.specFilter;
    jasmineEnv.configure({ specFilter: customSpecFilter });
```

Also, add the following `jasmine` peerDependency to package.json:

```
    "jasmine": "^3.3.0",
```

**Note:**

I'm happy to send a pull request with these changes; however, I believe I do not have permissions to do so at this time.

Thank you!

**Comment by mnasyrov** [comment]
Hi! Thank you for reporting about this, I will take a look.

**Comment by mnasyrov** [comment]
Fixed. Published `karma-jasmine-spec-tags@1.1.0` to npm.

**Comment by owenmead** [comment]
This fix broke some things, which I've submitted as a fix in https://github.com/mnasyrov/karma-jasmine-spec-tags/pull/4

**Comment by mnasyrov** [comment]
@owenmead Thanks for the fix. I published `karma-jasmine-spec-tags@1.2.0`
